src/Common/Select.php

```diff
@@ -612,12 +612,14 @@ public function groupBy(array $spec)
      *
      * @param string $cond The HAVING condition.
      *
+     * @param array ...$bind arguments to bind to placeholders
```

Comment:
OK. That makes sense. Declaring `@param string...$bind` is no problem (phpStorm complains, but it seems to be lagging behind the standard so I'm not too worried).
If we declare `function addCol($col, string ...$value)` then the minimum php version is 7.0. Is that OK?
Moving to 7.0 would let us do some other good stuff like `declare(strict_types=1)`